content/departments/product-engineering/engineering/onboarding/pr-checklist.md

```diff
@@ -0,0 +1,21 @@
+# Pull Request Guidelines Checklist
+
+### This page is to serve as a guideline checklist early career engineers or outside contributors can refer to when creating their first couple of PRs while developing for Sourcegraph.
+
+- Suggested branch naming convention: initials/description<br>
+  Example: "aa/replace-button-title"
+
+- As an engineering convention, name the PR with the team the change is associated with and a short but precist description of the change because once merged, this is typically how the commit gets its message, making it easier for anyone to broadly understand what area of impact a commit has when scanning commit history.<br>
+  Example: "batches: create batch change form"
+
+- Apply applicable team label to the PR so that the associated team can easily find and identify PRs at a glance
```

Comment:
Maybe include a screenshot or give an example like `team/batchers` so folks know how to search for those labels (by starting with "team/")